cmd/create.go

```diff
@@ -71,6 +72,18 @@ func buildInstance(options *options.Options) (*computepb.Instance, error) {
 	if err != nil {
 		return nil, err
 	}
+	serviceAccounts := []*computepb.ServiceAccount{}
+	if options.ServiceAccount != "none" {
```

Comment:
same as in the provider.yaml file, please use the empty string as the default case

Reply:
Thanks, fixed in https://github.com/loft-sh/devpod-provider-gcloud/pull/9/commits/ce4731d1041078ad7327efd0b57167dfdbfd290d